app/views/partial/_nav.html.erb

```diff
@@ -0,0 +1,17 @@
+<section class="container-fluid bg-primary p-2 mb-3">
+  <div class="row">
+    <div class="col-4">
+      <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="#fff" class="bi bi-list" viewBox="0 0 16 16">
+        <path fill-rule="evenodd" d="M2.5 12a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5zm0-4a.5.5 0 0 1 .5-.5h10a.5.5 0 0 1 0 1H3a.5.5 0 0 1-.5-.5z"/>
+      </svg>
+    </div>
+    <div class="col-6">
+      <h4 class="text-white">TRANSACTIONS</h4>
+    </div>
+    <div class="col-2">
+      <svg xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="#fff" class="bi bi-search" viewBox="0 0 16 16">
+        <path d="M11.742 10.344a6.5 6.5 0 1 0-1.397 1.398h-.001c.03.04.062.078.098.115l3.85 3.85a1 1 0 0 0 1.415-1.414l-3.85-3.85a1.007 1.007 0 0 0-.115-.1zM12 6.5a5.5 5.5 0 1 1-11 0 5.5 5.5 0 0 1 11 0z"/>
+    </svg>
+    </div>
+  </div>
+</section>
```

Comment:
- [ ] You don't need to display the sign-up or login on the home page after a user signs in successfully. You can add any information apart from the login or sign-up.

![image](https://user-images.githubusercontent.com/55635977/154803989-0ce2dda0-d33f-41a3-a2fa-c62402a43dfb.png)

Reply:
Thank you. issue resolved